### DISCUSSION TITLE
From fixed patern remove alpha background from # then send it to OCR

### DISCUSSION THREAD

**Discussion by DynV** [ImageMagick/ImageMagick#7129] [Help]
### ImageMagick version

7.0.10-61 Q16 x86_64 2023-05-13

### Operating system, version and so on

Cygwin 64 3.5.0-1

### Description

There's a plethora of images I'd like for an automated process to go through. Some of them have a specific pattern that contain a # which always start at the same place (and the end change depending on the #). All the #s in that pattern have a translucent background, which is always "tinted" the same color (the pattern and the # is bright green and the remainder of that part of the images is a darker green over an almost-always different underlying content); I'd like for that background to be cropped out then the image part containing the # sent to an OCR, which hopefully have an option to only look for a # (so won't recognize it as some weird characters).

At the end of the process I'd like to have a list of the file names containing the specific pattern and the result of the OCR, even if it's inconclusive (eg file000753.ext 57, file001573.ext N/A, file031726.ext 194). Is there a way to get that?

Thank you kindly

**Comment by fmw42** [comment]
It would help for anyone to assess your problem, if you post and example image and explain there what is fixed and what changes.

**Reply by DynV** [reply]
From 2 images I want a text file with the filename of
![20240224151808_1](https://github.com/ImageMagick/ImageMagick/assets/2823807/1bcda393-7de6-4abc-9fd0-b165fca43ba0)
then right after " 85, " then the filename of https://imgur.com/gaWmMoK then " 25.", if they were randomly placed in a directory along other images that doesn't have the same pattern, alike https://imgur.com/a/4c8745p Or if that's not possible I want images like the following from the attached above, but with better cropping:
<img width="37" alt="20240224151808_1-level_number" src="https://github.com/ImageMagick/ImageMagick/assets/2823807/d73fc9dc-92af-406c-80e7-a0926ac7f95c">
**Update 1:** Or instead of the small image, the green would be turned into black before being saved, something alike, but hopefully much better looking:
<img width="37" alt="20240224151808_1-level_number-blackened" src="https://github.com/ImageMagick/ImageMagick/assets/2823807/9413c8f5-e11f-461f-938e-d199c09810ee">
